src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -360,53 +399,37 @@ def value(self):
         """Get selected value"""
         return self._slider.value()
     
-    def spinBoxFrameRate(self) -> qt.QSpinBox:
-        """Returns the SpinBox widget for FrameRate display."""
-        return self._menuPlaySlider._framerate_action._spinbox
-
-    def spinBoxInterval(self) -> qt.QSpinBox:
-        """Returns the SpinBox widget for interval display."""
-        return self._menuPlaySlider._interval_action._spinbox
-
```

Comment:
Even if it is done for other widgets here, it is IMO not a good idea to expose internal widgets through the public API, it's best to provide the API to get/set the corresponding state.